src/main/resources/Message.properties

```diff
@@ -16,6 +16,7 @@ NO_TRANSIT_TIMES          = No transit times available. The date may be past or
 GEOCODE_FROM_NOT_FOUND    = Origin is unknown. Can you be a bit more descriptive?
 GEOCODE_TO_NOT_FOUND      = Destination is unknown.  Can you be a bit more descriptive?
 GEOCODE_FROM_TO_NOT_FOUND = Both origin and destination are unknown. Can you be a bit more descriptive?
+GEOCODE_INTERMEDIATE_NOT_FOUND = An intermediate destination is unknown. Can you be a bit more descriptive?.

```

Comment:
Is this file encoded properly? Since Java 9, Java properties files should be UTF-8. https://docs.oracle.com/javase/9/intl/internationalization-enhancements-jdk-9.htm#JSINT-GUID-9DCDB41C-A989-4220-8140-DBFB844A0FCA

Reply:
- All  properties files in this directory are properly encoded in UTF-8 (tested with iconv)
- Intellij needs to be configured to display properly UTF-8 properties file: Settings > Editor > Encoding > File Encodings > Default encoding for properties files
- Some properties file have Windows line separators, others have Linux/MacOs line separators. This does not seem to have any visible effect. These files can be normalized with Linux/MacOs line separator (this is the line separator used in the java source files) (can be done in another PR)